app.js

```diff
@@ -154,6 +154,17 @@ db.child('cases').on('value', (snap) => {
   }
 });
 
+// Watch inactive cases and open them up if volunteers have not done anything for over 1 hour
+setInterval(() => {
+  db.child('cases').once('value', (snap) => {
+    Object.keys(snap.val()).forEach((k) => {
+      if (Date.now() - snap.val()[k].last_modified > 60 * 60 * 1000)
```

Comment:
Moment is not very hard to add, but given that we only have one line like this, I didn't think we needed it. Minimizing the number of dependencies for each of our repositories is preferred.